cli/app.go

```diff
@@ -173,6 +174,32 @@ var app = &cli.App{
 					ArgsUsage: "[organization]",
 					Action:    ListLocationsAction,
 				},
+				{
+					Name:  "api-key",
+					Usage: "work with an api-key for your location",
+					Subcommands: []*cli.Command{
+						{
+							Name:  "create",
+							Usage: "create an api key for your location",
+							Flags: []cli.Flag{
+								&cli.StringFlag{
+									Name:     dataFlagLocationID,
+									Required: true,
+									Usage:    "The location to create an api-key for",
```

Comment:
```suggestion
									Usage:    "the location to create an api-key for",
```
Our usage texts are all lowercase IIRC. Here and for all usage texts below.

Reply:
removed this anyways